bundles/org.openhab.binding.pirateweather/src/main/resources/OH-INF/thing/bridge-types.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="pirateweather"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Pirate Weather Binding -->
+	<bridge-type id="weather-api">
+		<label>Pirate Weather Account</label>
+		<description>Provides access to the Pirate Weather API.</description>
+
```

Comment:
Please add semantic equipment tag.
For reference: https://next.openhab.org/docs/developer/bindings/thing-xml.html#thing-semantic-equipment-tags